klaviyo/public.py

```diff
@@ -63,8 +72,31 @@ def track(
         if ip_address:
             params['ip'] = ip_address
 
-        query_string = self._build_query_string(params, is_test)
-        return self._public_request(self.TRACK, query_string)
+        if method not in ['get','post']:
+
+            raise ValueError('method argument must be either "post" or "get"')
+
+        else:
```

Comment:
Nit: this else isn't really needed and removing it keeps things from getting too indented below